pipeline/runtime/iacm_nudge.go

```diff
@@ -0,0 +1,140 @@
+package runtime
+
+import (
+	"errors"
+	"github.com/harness/lite-engine/logstream"
+)
+
+func getIACMNudges() []logstream.Nudge {
+	return []logstream.Nudge{
+		logstream.NewNudge("[ERROR] approval step has been rejected",
```

Comment:
This should not look like an error. we can remove the "Check why...". 
Can we add the name of the user who perform the operation?

Reply:
We do not have that information in the logs at this stage so we can't print the user that rejected the step